great_expectations/expectations/metrics/map_metric_provider/map_condition_auxilliary_methods.py

```diff
@@ -830,3 +830,24 @@ def _spark_map_condition_query(
         "Column<'(", ""
     ).replace(")'>", "")
     return f"df.filter(F.expr({unexpected_condition_filtered}))"
+
+
+def _generate_temp_table_for_sql_map_condition(
```

Comment:
Helper function for generating `Table` object.

Reply:
Just a rename suggestion:

```suggestion
def _generate_temp_table(
```